features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -560,10 +560,15 @@
             {% else %}
             <UserInfoEndpointResponseBuilder>org.wso2.carbon.identity.oauth.endpoint.user.impl.UserInfoJSONResponseBuilder</UserInfoEndpointResponseBuilder>
             {% endif %}
-            {% if oauth.prompt_consent is sameas true %}
-            <SkipUserConsent>false</SkipUserConsent>
+            {% if oauth.prompt_login_consent is sameas true %}
+            <SkipLoginConsent>false</SkipLoginConsent>
             {% else %}
-            <SkipUserConsent>true</SkipUserConsent>
+            <SkipLoginConsent>true</SkipLoginConsent>
+            {% endif %}
+            {% if oauth.prompt_logout_consent is sameas true %}
+            <SkipLogoutConsent>false</SkipLogoutConsent>
+            {% else %}
+            <SkipLogoutConsent>true</SkipLogoutConsent>
```

Comment:
Don't we need these newly introduced configs in identity.xml file also https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml#L451